src/control/tools/StrokeHandler.cpp

```diff
@@ -184,14 +184,11 @@ void StrokeHandler::onButtonReleaseEvent(const PositionInputData& pos)
 	// Backward compatibility and also easier to handle for me;-)
 	// I cannot draw a line with one point, to draw a visible line I need two points,
 	// twice the same Point is also OK
-	if (stroke->getPointCount() == 1)
+	if (auto const& pv = stroke->getPointVector(); pv.size() == 1)
```

Comment:
Descriptive naming...

Reply:
Actually in this case, the most would be over descriptive. "pv" is a scope variable with less than 10 lines, and the assignment is more than descriptive. I follow the guideline, if it can be inlined as good as using a temporary and if the scope is short(up to 15+-5 lines), use a short variable name. The use of this temporary is there to reduce duplicate calls to one long function.